packages/shared/src/libs/helpers/debounce/debounce.helper.ts

```diff
@@ -0,0 +1,31 @@
+function debounce<T extends unknown[]>(
```

Comment:
```suggestion
function initDebounce<T extends unknown[]>(
```
do not forget about file name